README.rst

```diff
@@ -85,6 +85,32 @@ directory you can do this::
 If there's not output. It worked. Check how it edited your
 requirements files.
 
+Filtering requirements by Python version
```

Comment:
How about "Filtering releases by Python version"? I think that matches up more closely with the code's behavior. Hashes are just numbers, and don't have any particular relation to a Python version.